Sources/Extensions/APIGatewayExtensions.swift

```diff
@@ -7,13 +7,13 @@ private let jsonEncoder = JSONEncoder()
 
 extension APIGatewayV2Request {
     
-    public func decode<C: Codable>(as type: C.Type = C.self) throws -> C {
+    public func decode<D: Decodable>(as type: D.Type = D.self) throws -> D {
         guard let body = self.body,
               let dataBody = body.data(using: .utf8)
```

Comment:
Please prefer `Data(body.utf8View)` for getting `Data` from a `String` (and for going in the other direction, prefer `String(decoding: data, as: UTF8.self)`).